fetch.bs

```diff
@@ -6710,6 +6748,362 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+<p>The <dfn>minimal subframe deferred-fetch quota</dfn> is 8 kibibytes.
+<p>The <dfn>deferred-fetch delegated quota</dfn> is 128 kibibytes.
+
+<div algorithm>
+<p>Two {{Document}}s <var>a</var> and <var>b</var> are said to
+<dfn>share deferred-fetch quota</dfn> if <var>a</var>'s
+<a>relevant agent</a> is <var>b</var>'s <a>relevant agent</a>, and <var>a</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>b</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a>.
+</div>
+
+
+<div algorithm>
+<p>To get the <dfn>available deferred-fetch quota</dfn> given a {{Document}}
+<var>document</var> and an <a for=/>origin</a>-or-null <var>origin</var>:
+
+<ol>
+ <li><p>Let <var>quota</var> be zero.
+
+ <li>
+  <p>Let <var>quotaForOrigin</var> be 64 kibibytes.
+  <p class=note>
+
+ <li>
+  <p>For each <var>otherNavigable</var> in <var>document</var>'s <a>node navigable</a>'s
+  <a for=navigable>top-level traversable</a>'s <a>inclusive descendant navigables</a>:
+
+  <p class=note>This algorithm iterates over the entire navigable tree. It accumulates quota from
+  the <a for=/>top-level traversable</a>, and from subframes who inherit quota from cross-origin or
+  cross-agent navigables. Subsequently, it subtracts the quota delegated to cross-origin or
+  cross-agent subframes, as well as quota spent on pending deferred fetch requests.
+
+  <ol>
+   <li><p>Let <var>otherDocument</var> be <var>otherNavigable</var>'s <a>active document</a>.
+
+   <li>
+    <p>If <var>document</var> and <var>otherDocument</var> <a>share deferred-fetch quota</a>,
+    then:
+    <ol>
+     <li>
+      <p>If <var>otherDocument</var>'s <a>node navigable</a> is a
+      <a for=/>top-level traversable</a>, then:
+
+      <p class=note>Initialize the quota from the top-level traversable.
+
+      <ol>
+       <li><p>Assert: <var>quota</var> is zero.
+
+       <li><p>If <var>otherDocument</var> is not
+       <a>allowed to use</a> the <a>policy-controlled feature</a>
+       {{PermissionPolicy/"deferred-fetch"}}, then return 0.
+
+       <li>
+        <p>Set <var>quota</var> be 640 kibibytes.
+        <p class="note allow-2119">640kb should be enough for everyone.
+
+       <li><p>If <var>otherDocument</var> is <a>allowed to use</a> the
+       <a>policy-controlled feature</a> {{PermissionPolicy/"deferred-fetch-minimal"}}, then
+       decrement <var>quota</var> by <a>deferred-fetch delegated quota</a>.
+      </ol>
+
+     <li>
+      <p>Otherwise,
+
+      <p class=note>This is a subframe, check if it inherits quota from its parent container based
+      on permissions policy. This would only be the case if the subframe and its parent do not share
+      quota, as quota is either shared or delegated.
+
+      <ol>
+       <li><p>Let <var>otherContainer</var> be <var>otherNavigable</var>'s
+       <a>navigable container</a>.
+
+       <li>
+        <p>If <var>otherContainer</var>'s <a>node document</a> and <var>otherDocument</var> do
```

Comment:
```
Let otherContainer be otherNavigable’s navigable container.

If otherContainer’s node document and otherDocument do not share deferred-fetch quota, then:
```

* Combing the above two steps, does `otherContainernode document` just mean `otherNavigable`?
  * If so, why not just say `otherNavigable` directly?
  * If so, does `If otherContainer’s node document and otherDocument do not share deferred-fetch quota, then:` checking the quota against the same document?

Reply:
No,  `otherContainer` `node document` is `otherNavigable`'s parent